### PR TITLE
Add trading_name to global search.

### PR DESCRIPTION
**Pull request by elcct**
As in the title, it adds `trading_name` (alias for `alias` :) ) to the global search. It also has a small cleanup of tests and small improvement to queries.

### CODE REVIEW

File: datahub/search/company/test/test_elasticsearch.py [hunk 1]
```diff
@@ -14,35 +14,29 @@ def test_get_basic_search_query():
...
                         }
                     }, {
                         'match': {
                             'address_1': 'test'
                         }
                     }, {
                         'match': {
-                            'address_2': 'test'
-                        }
+                            'address_2': 'test'}
```
Comment (marcofucci) [inline datahub/search/company/test/test_elasticsearch.py]: You might have forgotten to indent the closing bracket

Comment (elcct) [reply datahub/search/company/test/test_elasticsearch.py]: Thanks, updated.

**Comment by elcct** [comment]
@reupen what do you think about changing model's field from `alias` to `trading_name` ?

**Comment by reupen** [comment]
> @reupen what do you think about changing model's field from alias to trading_name ?

Yeah, that seems fair, probably better as a separate PR though.

**Pull request merged**